include/crocoddyl/multibody/frames.hpp

```diff
@@ -137,7 +137,7 @@ struct FrameFrictionConeTpl {
 };
 
 template <typename _Scalar>
-class FrameCoPSupportTpl {
+struct FrameCoPSupportTpl {
```

Comment:
Revert it back. This is a class became we need to protect the wrong use of some internal members, there is a reason for this. (Please keep focus in the goal of your PR)

Reply:
The "goal" of this PR doesn't work, if there are some bad changes already introduced in the `devel` that produce errors. `FrameCoPSupport` has been forward declared as a struct, like _ALL_ other frames. Be consistent.

Reply:
This is not a bad change. Take time to understand the reason of this choice. However, in short, we need to have private members in order to avoid user mistakes.

The other frameXXX don't need to make private members.

---
Consider that if you modify "bad" (or not) changes that don't correspond to the main goal of a PR, then it makes my job more difficult and slow since I need to pay attention to each line.

Typically, you made huge PRs. This is not good / safe for the project. There many things that I might miss, and it is the same for you. Then, in future, when I track merged PRs, I might lost these bits.

It is easy for you to make independent PRs when you found things that you believe we should change. I would be happy if you make that effort for me :)

Reply:
Carlos, Please first read what my comment was, before replying.

`FrameCopSupport`, like all other frames, has been forward declared as a `struct` here: https://github.com/loco-3d/crocoddyl/blob/devel/include/crocoddyl/multibody/fwd.hpp#L78. Re-declaring it as a `class` produces ABI errors. I think it is better to keep this as a struct, like all other frames, and define the private and public members explicitly, as is being done.

> It is easy for you to make an independent PR when you found things that you believe we should change

Implementation errors are not always avoidable. However many of your PRs in crocoddyl are merged really quickly without any review _at all_. This introduces a lot of them, and it is diffcult to continuously branch everytime I find a problem. This was a minor issue, and the change proposed was _consistent with the API_.

Finally, I like 50% of your reviews, which often provide constructive remarks. However, starting a review with "Revert it back". or "Remove this.", when you don't understand what the change is, is counter-productive and leads to frivolous debates like this. Let's avoid that in the future :).